### ISSUE TITLE
Auto counter offers.

### ISSUE THREAD

**Issue by angelleye** [angelleye/offers-for-woocommerce#339]
Right now we have the ability to auto-accept or auto-decline offers, but users are requesting the ability to setup auto-counter rules as well.  

So we'd have to plan on how to set that up.  What options to give the user and what logic would be necessary to ensure the highest possible price for the seller in the end based on their rules.

**Comment by digitsoft** [comment]
I love this idea!  We did this 10 years ago for an online automobile startup I helped create.
The rules and code for this may actually be prohibitive time wise - it can get complex.

Example Rules: 
if the buyer offer is less than half of the item price
AND
if this is the buyer's first offer
THEN
counter offer with item price minus 10%


if the buyer offer is less than half of the item price
AND
if this is not the buyer's first offer
THEN
counter offer with item price minus 15%
AND
set offer expires 72 hours from today

if the buyer offer is less than half of the item price
AND
if this is not the buyer's first offer
AND
if the buyer offer is $1 more than the last buyer offer
THEN
counter offer with item price minus 15%
AND
set offer expires 72 hours from today
AND
set to final offer
